src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -1524,36 +1528,55 @@ private IOperation CreateBoundLocalDeclarationOperation(BoundLocalDeclaration bo
                     // var statement points to VariableDeclarationSyntax
                     varStatement = node.Parent;
 
+                    varDeclaration = node.Parent;
+
                     // var declaration points to VariableDeclaratorSyntax
```

Comment:
>VariableDeclarationGroupOperation [](start = 23, length = 33)

Given the current model, I think we should create IVariableDeclarationGroupOperation node only for LocalDeclarationStatementSyntax. For all other cases we should simply return IVariableDeclarationOperation associated with VariableDeclarationSyntax. As an added benefit, we won't need to play tricks with implicit state.

Reply:
I'm thinking that, for consistency with VB, especially in the IUsingOperation tree, we should leave this there. However, I will update this scenario to mark the Group as the implicit node, with the Declaration as a explicit one.

---
In reply to: [147783057](https://github.com/dotnet/roslyn/pull/22842#discussion_r147783057) [](ancestors = 147783057)